mffpy/tests/test_writer.py

```diff
@@ -35,11 +35,12 @@ def test_writer_receives_bad_init_data():
 
 
 def test_writer_doesnt_overwrite(tmpdir):
-    """test that `mffpy.Writer` doesn't overwrite existing files"""
+    """test that `mffpy.Writer` doesn't overwrite existing files by default"""
     dirname = join(str(tmpdir), 'testdir.mff')
     makedirs(dirname, exist_ok=True)
     with pytest.raises(AssertionError, match='File.*exists already'):
         Writer(dirname)
+    Writer(dirname, overwrite=True)
```

Comment:
Let's make a new test for overwriting an existing MFF. It should

1. Check that the appropriate files are overwritten
2. Check that extra files are removed

This could look pretty similar to function `test_writer_writes`, but with an extra step to overwrite the first file written.